src/distributors/distributor-factory.ts

```diff
@@ -1,13 +1,14 @@
-import { AdoptOpenJDKDistributor } from './adoptopenjdk-installer';
+import { AdoptiumDistributor } from './adoptium/adoptium-installer';
```

Comment:
I think we are "overnaming" files now. Since we specify folder `adoptium`, we can use name `installer` without specifying adoptium once again.
so we can have structure
```
distributors
  - adoptium
     - installer
     - models
  - zulu
     - installer
     - models
  - another vendors goes here
```

Reply:
Done. Thank you.